pom.xml

```diff
@@ -23,7 +23,7 @@
   </parent>
 
   <artifactId>vertx-core</artifactId>
-  <version>3.5.0-SNAPSHOT</version>
+  <version>3.5.0-SNAPSHOT-CMP</version>
```

Comment:
No, I had been using that marker to differentiate the modified version from the master in my fork. I'll remove it.